onnxruntime/core/framework/random_seed.h

```diff
@@ -12,10 +12,9 @@ namespace utils {
  * Gets the static random seed value. All calls to GetStaticRandomSeed()
```

Comment:
this documentation needs updating. also would suggest renaming to Get[Global?]RandomSeed()